modules/account-lib/test/unit/coin/near/transactionBuilder/stakingActivateBuilder.ts

```diff
@@ -0,0 +1,93 @@
+import should from 'should';
+import * as testData from '../../../../resources/near';
+import { register } from '../../../../../src';
+import { TransactionBuilderFactory } from '../../../../../src/coin/near';
+import { TransactionType } from '../../../../../src/coin/baseCoin';
+import { validatorContractAddress } from '../../../../resources/near';
+
+describe('Near Staking Activate Builder', () => {
+  const factory = register('tnear', TransactionBuilderFactory);
+  const gas = '125000000000000';
+  const amount = '1000000'; // amount to be staked in yoctos
+
+  describe('Succeed', () => {
+    it('build a create and delegate staking signed tx', async () => {
+      const txBuilder = factory.getStakingActivateBuilder();
+      txBuilder
+        .amount(amount)
+        .gas(gas)
+        .sender(testData.accounts.account1.address, testData.accounts.account1.publicKey)
+        .receiverId(validatorContractAddress)
+        .recentBlockHash(testData.blockHash.block1)
+        .nonce(1);
+      txBuilder.sign({ key: testData.accounts.account1.secretKey });
+      const tx = await txBuilder.build();
+      tx.inputs.length.should.equal(1);
+      tx.inputs[0].should.deepEqual({
+        address: testData.accounts.account1.address,
+        value: amount,
+        coin: 'tnear',
+      });
+      tx.outputs.length.should.equal(1);
+      tx.outputs[0].should.deepEqual({
+        address: validatorContractAddress,
+        value: amount,
+        coin: 'tnear',
+      });
+
+      should.equal(tx.type, TransactionType.StakingActivate);
+      const rawTx = tx.toBroadcastFormat();
+      should.equal(rawTx, testData.rawTx.stakingActivate.signed);
```

Comment:
I think you should check tx.toJson(), particularly actions.

Reply:
good catch!